docs/rules/max-classes-per-file.md

```diff
@@ -0,0 +1,43 @@
+# enforce a maximum number of classes per file (max-classes-per-file)
+
+Ensures that each file has a single purpose and holds responsibility
+for only one class. Often a file containing several classes should
+most likely be separated into multiple for cleanliness.
+
+## Rule Details
+
+This rule enforces that each file may contain only a particular number
+of classes and no more.
+
+Examples of **incorrect** code for this rule:
+
+```js
+/*eslint max-classes-per-file: ["error", 1]*/
```

Comment:
It would be good to have examples with the rule set to error with no options (which I assume is equivalent to `["error", 1]`). And you could move the examples which set an explicit number into the Options section, per my note there.